app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
```

Comment:
It's not a good design to simply return `None` when user will try to add unacceptable data types - it should raise an exception to signal something went wrong. You may create custom exceptions to process this error but in this task is better just to simplify this method to have three lines of code - one if statement and two return statements.
Check your other methods where you return `None` and change accordingly